File: api/src/main/java/club/yunzhi/smartcommunity/entity/Community3d.java
package club.yunzhi.smartcommunity.entity;

import org.hibernate.annotations.SQLDelete;
;

import javax.persistence.Entity;

@SQLDelete(sql = "update `community3d` set deleted = 1 where id = ?")
@Entity
public class Community3d extends SoftDeleteEntity {
  private String name;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }
}

File: web/projects/lib/src/entity/community3d.ts
/**
 * 社区3D模型实体
 */
export class Community3d {
  /**
   * id
   */
  id: number;

  /**
   * 名称
   */
  name: string;

  constructor(data = {} as {
    id?: number,
    name?: string
  }) {
    this.id = data.id;
    this.name = data.name;
  }
}
